src/components/StylePopup/StylePopup.js

```diff
@@ -116,7 +144,7 @@ class StylePopup extends React.PureComponent {
           onMouseDown={e => e.preventDefault()}
         >
           <div className="sliders">
-            {!this.props.hideSlider && this.renderSliders()}
+            {!hideAllSlider && this.renderSliders()}
```

Comment:
Maybe have this conditional rendering around `<div className="sliders-container ..." >`, just to match your other code?

Reply:
addressed